recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -12,43 +11,53 @@
 
 class BehaviorTreeCPPConan(ConanFile):
     name = "behaviortree.cpp"
+
     description = "This C++ library provides a framework to create BehaviorTrees"
     license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/BehaviorTree/BehaviorTree.CPP"
     topics = ("ai", "robotics", "games", "coordination")
+
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
         "with_tools": [True, False],
         "with_coroutines": [True, False],
+        "with_groot_interface": [True, False],
+        "with_sqlite_logging": [True, False],
```

Comment:
If these are specific to a version, they should be removed from config_options or package ID